tests/unit_tests/blas/batch/gemm_batch_stride.cpp

```diff
@@ -213,29 +282,49 @@ class GemmBatchStrideTests
         : public ::testing::TestWithParam<std::tuple<sycl::device *, oneapi::mkl::layout>> {};
 
 TEST_P(GemmBatchStrideTests, RealHalfPrecision) {
-    EXPECT_TRUEORSKIP(test<sycl::half>(std::get<0>(GetParam()), std::get<1>(GetParam()), 5));
+    EXPECT_TRUEORSKIP((test<sycl::half, sycl::half, sycl::half, sycl::half>(
+        std::get<0>(GetParam()), std::get<1>(GetParam()), 5)));
+}
+
+TEST_P(GemmBatchStrideTests, RealHalfRealScalarPrecision) {
+    EXPECT_TRUEORSKIP((test<sycl::half, sycl::half, float, float>(std::get<0>(GetParam()),
+                                                                  std::get<1>(GetParam()), 5)));
+}
+
+TEST_P(GemmBatchStrideTests, RealIntRealScalarPrecision) {
```

Comment:
```suggestion
TEST_P(GemmBatchStrideTests, Int8Int8SinglePrecision) {
```
more like what is in `gemm_bias` tests. (Single could also be Float32, not sure what is best).